src/components/cell-control/autocompleter.md

```diff
@@ -6,22 +6,22 @@ import { Table, TableHeader, TableHeaderCell, TableBody, TableRow, TableCell } f
 
 const ids = {
   th: {
-    autocompleter: uuid.v4(),
+    example: uuid.v4(),
```

Comment:
I fixed a bunch of bad IDs in my recent examples. I made it more copy-pastable for now. Ideally, we should have code that _verifies_ the existence of labels. But that's another problem to solve in the future.